### ISSUE TITLE
Helicopter infinite audio (MP)

### ISSUE THREAD

**Issue by Skaidri** [TEHE-Studios/ExpandedHelicopterEvents#358]
**Describe the bug**

Helicopters/Jets audio won't stop after it's gone. Audio can stack if multiple helis/jets fly by in on session. Only server restart helps.

**To Reproduce**
Just wait for another heli fly by on MP mode

**Additional context**
It doesn't happen on singleplayer

**Comment by joveice** [comment]
Am experiencing this as well.

**Comment by 6942069420-del** [comment]
Happens to us too

**Comment by Nerdgamerdave** [comment]
All players log out.  Reset Server.  Log back in.  Should fix.  I think it's related to desync.  Some MP servers have zero issues with this, others do, so I myself think it's related to mods that are maybe conflicting with EHE stuff and cause the audio to stick.

You'll notice in -debug tho that the audio is OFF.  It's not drawing zeds or making any actual in game noise.  Sound is just "stuck"

**Comment by Chuckleberry-Finn** [comment]
Indeed, it's been an on-going issue and would require a bit of a rewrite. May have to do with chunkloading losing track of sounds from so far away.

**Comment by 6942069420-del** [comment]
Once you have this fixed I will be happy to add this mod back into the RegularZomboid server, It's amazing  but this one issue messes us up lol

**Comment by Chuckleberry-Finn** [comment]
Currently Testing some changes/fixes: https://steamcommunity.com/sharedfiles/filedetails/?id=3171846102

Lmk if you come into more issues.

**Comment by 6942069420-del** [comment]
The issue still happens with the Beta version of EHE

It happened with a Green jet flyby

It created this awful roaring sound that you could only hear on your left ear

Along with the non glitched version of the jet sound you could actually hear flying by

Clients relogging fixed the issue

> Currently Testing some changes/fixes: https://steamcommunity.com/sharedfiles/filedetails/?id=3171846102
> 
> Lmk if you come into more issues.

**Comment by evheniydan** [comment]
Any updates on this issue? The sound of a lagging jet is irritating and I need to restart the server every time to fix it.

**Comment by Chuckleberry-Finn** [comment]
dupe of #302
